From f9b2eda98a9fcd5e8e3f2308e9e9824ad865ce01 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Sat, 28 Sep 2019 20:57:18 +0200
Subject: [PATCH 01/15] ENH: Fix path to install
 ModuleDescriptionParserConfig.cmake

---
 ModuleDescriptionParser/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ModuleDescriptionParser/CMakeLists.txt b/ModuleDescriptionParser/CMakeLists.txt
index 790c789..6a2769a 100644
--- a/ModuleDescriptionParser/CMakeLists.txt
+++ b/ModuleDescriptionParser/CMakeLists.txt
@@ -216,7 +216,7 @@ if(NOT DEFINED ${PROJECT_NAME}_INSTALL_BIN_DIR)
 endif()
 
 if(NOT DEFINED ${PROJECT_NAME}_INSTALL_LIB_DIR)
-  set(${PROJECT_NAME}_INSTALL_LIB_DIR lib/${PROJECT_NAME})
+  set(${PROJECT_NAME}_INSTALL_LIB_DIR lib64/${PROJECT_NAME})
 endif()
 
 install(TARGETS ${lib_name}
@@ -239,7 +239,7 @@ if(NOT ${PROJECT_NAME}_INSTALL_NO_DEVELOPMENT)
   install(FILES
     ${CMAKE_CURRENT_BINARY_DIR}/UseModuleDescriptionParser.cmake
     ${CMAKE_CURRENT_BINARY_DIR}/install/ModuleDescriptionParserConfig.cmake
-    DESTINATION lib/${lib_name} COMPONENT Development)
+    DESTINATION lib64/${lib_name} COMPONENT Development)
 endif()
 
 # --------------------------------------------------------------------------
-- 
2.24.1

