From 97cb9428fc99b97dcb5d0430ba31141bbdb91b11 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Sat, 28 Sep 2019 22:29:13 +0200
Subject: [PATCH 05/15] ENH: Generate TCLAP install configuration files

---
 tclap/GenerateTCLAPConfig.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tclap/GenerateTCLAPConfig.cmake b/tclap/GenerateTCLAPConfig.cmake
index aaa62a0..490c85d 100644
--- a/tclap/GenerateTCLAPConfig.cmake
+++ b/tclap/GenerateTCLAPConfig.cmake
@@ -14,7 +14,7 @@ set(TCLAP_USE_FILE_CONFIG ${TCLAP_BINARY_DIR}/UseTCLAP.cmake)
 
 # Configure TCLAPConfig.cmake for the install tree.
 configure_file(${TCLAP_SOURCE_DIR}/TCLAPConfig.cmake.in
-               ${TCLAP_BINARY_DIR}/TCLAPConfig.cmake @ONLY)
+              ${TCLAP_BINARY_DIR}/TCLAPConfig.cmake @ONLY)
 
 
 
@@ -24,5 +24,5 @@ configure_file(${TCLAP_SOURCE_DIR}/TCLAPConfig.cmake.in
 
 
 # TODO- Configure TCLAPInstallConfig.cmake for the install tree.
-#configure_file(${TCLAP_SOURCE_DIR}/TCLAPInstallConfig.cmake.in
-#               ${TCLAP_BINARY_DIR}/install/TCLAPConfig.cmake @ONLY)
+configure_file(${TCLAP_SOURCE_DIR}/TCLAPInstallConfig.cmake.in
+               ${TCLAP_BINARY_DIR}/install/TCLAPConfig.cmake @ONLY)
-- 
2.24.1

