From 9486281bd6e077c8279ea936358645b2a7ba0302 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Mon, 16 Dec 2019 06:06:42 +0100
Subject: [PATCH] COMP: Add the SlicerExecutionModel_INSTALL_LIB_DIR variable

---
 SlicerExecutionModelInstallConfig.cmake.in | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/SlicerExecutionModelInstallConfig.cmake.in b/SlicerExecutionModelInstallConfig.cmake.in
index c81dac1..7e2adf2 100644
--- a/SlicerExecutionModelInstallConfig.cmake.in
+++ b/SlicerExecutionModelInstallConfig.cmake.in
@@ -63,8 +63,11 @@ set(SlicerExecutionModel_LIBRARIES
 set(DEFAULT_SEM_TARGET_LIBRARIES
   @DEFAULT_SEM_TARGET_LIBRARIES_CONFIG@
   )
+set(SlicerExecutionModel_INSTALL_LIB_DIR
+"@SlicerExecutionModel_INSTALL_LIB_DIR@"
+)
 set(SlicerExecutionModel_USE_FILE
-  "@SlicerExecutionModel_INSTALL_LIB_DIR@/UseSlicerExecutionModel.cmake"
+  "${SlicerExecutionModel_INSTALL_LIB_DIR}/UseSlicerExecutionModel.cmake"
   )
 set(SlicerExecutionModel_CMAKE_DIR ${SlicerExecutionModel_INSTALL_LIB_DIR}/CMake)
 
-- 
2.24.1

