From e84107dc14a9b5744520afc0f8e1559681e6e125 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 17 Dec 2019 06:57:36 +0100
Subject: [PATCH 12/15] COMP: Fix UseGenerateCLP

---
 GenerateCLP/UseGenerateCLP.cmake.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/GenerateCLP/UseGenerateCLP.cmake.in b/GenerateCLP/UseGenerateCLP.cmake.in
index cc29133..0be3060 100644
--- a/GenerateCLP/UseGenerateCLP.cmake.in
+++ b/GenerateCLP/UseGenerateCLP.cmake.in
@@ -8,7 +8,7 @@ include(${TCLAP_USE_FILE})
 #
 # ModuleDescriptionParser
 #
-find_package(ModuleDescriptionParser REQUIRED)
+find_package(SlicerExecutionModel REQUIRED ModuleDescriptionParser)
 include(${ModuleDescriptionParser_USE_FILE})
 
 if(NOT ITK_FOUND) #Do not clobber existing found version of ITK
@@ -31,4 +31,4 @@ if(GenerateCLP_USE_JSONCPP)
   include_directories(${JsonCpp_INCLUDE_DIR})
 endif()
 
-include(${GenerateCLP_CMAKE_DIR}/GenerateCLP.cmake)
+include(${GenerateCLP_CONFIG_DIR}/GenerateCLP.cmake)
-- 
2.24.1

