From 9fa37c97b48a42986f62a917a75174563e80148d Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 15 Jan 2020 07:13:36 +0100
Subject: [PATCH] COMP: Fix path to SEMCommandLineLibraryWrapper.cxx

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ecb4a32..24d495a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -130,7 +130,7 @@ endif()
 #-----------------------------------------------------------------------------
 if(NOT DEFINED SlicerExecutionModel_DEFAULT_CLI_LIBRARY_WRAPPER_CXX)
   set(SlicerExecutionModel_DEFAULT_CLI_LIBRARY_WRAPPER_CXX
-    ${CMAKE_CURRENT_SOURCE_DIR}/CMake/SEMCommandLineLibraryWrapper.cxx)
+    ${SlicerExecutionModel_INSTALL_LIB_DIR}/CMake/SEMCommandLineLibraryWrapper.cxx)
 endif()
 
 #-----------------------------------------------------------------------------
-- 
2.24.1

