From 6f837894054dd0b3d6c2f83efd963039ee9a5a2c Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 16 Jan 2020 14:26:21 +0100
Subject: [PATCH] COMP: Add INSTALL_RPATH property to CLI exes (to be able to
 load .so)

---
 CMake/SEMMacroBuildCLI.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/SEMMacroBuildCLI.cmake b/CMake/SEMMacroBuildCLI.cmake
index ea1b4dc..c2fffd1 100644
--- a/CMake/SEMMacroBuildCLI.cmake
+++ b/CMake/SEMMacroBuildCLI.cmake
@@ -158,6 +158,7 @@ macro(SEMMacroBuildCLI)
 
   add_executable(${CLP} ${LOCAL_SEM_CLI_LIBRARY_WRAPPER_CXX})
   set_target_properties(${CLP} PROPERTIES COMPILE_FLAGS "${cli_executable_compile_flags}")
+    set_target_properties(${CLP} PROPERTIES INSTALL_RPATH "${LOCAL_SEM_INSTALL_RUNTIME_DESTINATION}")
   set(cli_executable_libraries ${CLP}Lib)
   if(DEFINED SlicerExecutionModel_EXTRA_EXECUTABLE_TARGET_LIBRARIES)
     list(APPEND cli_executable_libraries ${SlicerExecutionModel_EXTRA_EXECUTABLE_TARGET_LIBRARIES})
-- 
2.24.1

