diff --git a/Wrapping/Python/CMakeLists.txt b/Wrapping/Python/CMakeLists.txt
index 3466c412..f4da507b 100644
--- a/Wrapping/Python/CMakeLists.txt
+++ b/Wrapping/Python/CMakeLists.txt
@@ -82,13 +82,13 @@ if(DEFINED SKBUILD)
       ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK.py
       ${CMAKE_CURRENT_SOURCE_DIR}/Packaging/__init__.py
       ${SimpleITK_DOC_FILES}
-    DESTINATION SimpleITK
+    DESTINATION ${SimpleITK_INSTALL_LIBRARY_DIR}/SimpleITK
     COMPONENT Runtime
     )
 
   install(TARGETS ${SWIG_MODULE_SimpleITKPython_TARGET_NAME}
-    RUNTIME DESTINATION SimpleITK COMPONENT Runtime
-    LIBRARY DESTINATION SimpleITK COMPONENT Runtime
+    RUNTIME DESTINATION ${SimpleITK_INSTALL_LIBRARY_DIR}/SimpleITK COMPONENT Runtime
+    LIBRARY DESTINATION ${SimpleITK_INSTALL_LIBRARY_DIR}/SimpleITK COMPONENT Runtime
     )
 else()
   include(LegacyPackaging.cmake)
