From 815e22bad78e18c3413a0ccbdd8eeb7591dd8be9 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 25 Mar 2020 20:39:59 +0100
Subject: [PATCH 3/4] ENH: Adding install directories

---
 CMakeLists.txt | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c95a7fa..e137ea2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -168,6 +168,15 @@ endif()
 if(NOT DEFINED ${PROJECT_NAME}_INSTALL_LIB_DIR)
   set(${PROJECT_NAME}_INSTALL_LIB_DIR lib/${PROJECT_NAME})
 endif()
+ if(NOT DEFINED ${PROJECT_NAME}_INSTALL_PYTHON_MODULE_LIB_DIR)
+  set(${PROJECT_NAME}_INSTALL_PYTHON_MODULE_LIB_DIR ${${PROJECT_NAME}_INSTALL_LIB_DIR})
+endif()
+if(NOT DEFINED ${PROJECT_NAME}_INSTALL_PYTHON_LIB_DIR)
+  set(${PROJECT_NAME}_INSTALL_PYTHON_LIB_DIR ${${PROJECT_NAME}_INSTALL_LIB_DIR})
+endif()
+if(NOT DEFINED ${PROJECT_NAME}_INSTALL_CMAKE_DIR)
+  set(${PROJECT_NAME}_INSTALL_CMAKE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/CMake)
+endif()
 
 install(TARGETS ${lib_name}
   RUNTIME DESTINATION ${${PROJECT_NAME}_INSTALL_BIN_DIR} COMPONENT RuntimeLibraries
@@ -185,7 +194,8 @@ if(vtkAddon_WRAP_PYTHON)
     KIT_NAME ${lib_name}
     KIT_SRCS ${vtkAddon_SRCS}
     KIT_INSTALL_BIN_DIR ${${PROJECT_NAME}_INSTALL_BIN_DIR}
-    KIT_INSTALL_LIB_DIR ${${PROJECT_NAME}_INSTALL_LIB_DIR}
+    KIT_INSTALL_LIB_DIR ${${PROJECT_NAME}_INSTALL_PYTHON_LIB_DIR}
+    KIT_MODULE_INSTALL_LIB_DIR ${${PROJECT_NAME}_INSTALL_PYTHON_MODULE_LIB_DIR}
     )
   # Export target
   export(TARGETS ${lib_name}Python ${lib_name}PythonD APPEND FILE ${${PROJECT_NAME}_EXPORT_FILE})
-- 
2.24.1

