From 99c710adcc2def89be33c20c6253b47c522719a2 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 25 Mar 2020 20:40:42 +0100
Subject: [PATCH 4/4] ENH: Adding installation of configuration files

---
 CMakeLists.txt | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e137ea2..37b2f23 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -230,3 +230,12 @@ configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/vtkAddonConfig.cmake.in
   ${CMAKE_CURRENT_BINARY_DIR}/vtkAddonConfig.cmake
   )
+
+# --------------------------------------------------------------------------
+# Install configuration files
+# --------------------------------------------------------------------------
+install(
+  FILES ${CMAKE_CURRENT_BINARY_DIR}/vtkAddonConfig.cmake
+  DESTINATION ${${PROJECT_NAME}_INSTALL_CMAKE_DIR}
+  )
+
-- 
2.24.1

