From 2d6b8eda3a922ebd719f3952e691e20fe9278ec6 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Tue, 14 Mar 2023 16:48:05 +0100
Subject: [PATCH] ENH: Make ImportItkSnapLabel a separate module

---
 Modules/Scripted/ImportItkSnapLabel/CMakeLists.txt | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Modules/Scripted/ImportItkSnapLabel/CMakeLists.txt b/Modules/Scripted/ImportItkSnapLabel/CMakeLists.txt
index d4aa17093a..4d92d2ca48 100644
--- a/Modules/Scripted/ImportItkSnapLabel/CMakeLists.txt
+++ b/Modules/Scripted/ImportItkSnapLabel/CMakeLists.txt
@@ -1,6 +1,15 @@
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+#-----------------------------------------------------------------------------
+project(ImportItkSnapLabel)
+
 #-----------------------------------------------------------------------------
 set(MODULE_NAME ImportItkSnapLabel)
 
+#-----------------------------------------------------------------------------
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
+
 #-----------------------------------------------------------------------------
 set(MODULE_PYTHON_SCRIPTS
   ${MODULE_NAME}.py
-- 
2.39.2

