From db3e53876f8e27a5732193cd4894bb436ba5bc82 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Tue, 21 Feb 2023 10:06:10 +0100
Subject: [PATCH] ENH: Make ACPCTransform a separate module

---
 Modules/CLI/ACPCTransform/CMakeLists.txt | 22 ++++++++++++++++++++--
 1 file changed, 20 insertions(+), 2 deletions(-)

diff --git a/Modules/CLI/ACPCTransform/CMakeLists.txt b/Modules/CLI/ACPCTransform/CMakeLists.txt
index b938e5bb83..fde200d8fa 100644
--- a/Modules/CLI/ACPCTransform/CMakeLists.txt
+++ b/Modules/CLI/ACPCTransform/CMakeLists.txt
@@ -1,7 +1,14 @@
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+#-----------------------------------------------------------------------------
+project(ACPCTransform)
 
 #-----------------------------------------------------------------------------
 set(MODULE_NAME ACPCTransform)
 
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
+
 #-----------------------------------------------------------------------------
 
 #
@@ -10,6 +17,8 @@ set(MODULE_NAME ACPCTransform)
 find_package(SlicerExecutionModel REQUIRED)
 include(${SlicerExecutionModel_USE_FILE})
 
+link_directories(${Slicer_INSTALL_QTLOADABLEMODULES_LIB_DIR})
+
 #-----------------------------------------------------------------------------
 SEMMacroBuildCLI(
   NAME ${MODULE_NAME}
@@ -25,14 +34,23 @@ SEMMacroBuildCLI(
     ${VTK_LIBRARIES}
     vtkSlicerMarkupsModuleMRML
   INCLUDE_DIRECTORIES
+    ${Slicer_Base_INCLUDE_DIRS}
     ${vtkTeem_INCLUDE_DIRS}
-    ${MRMLCore_INCLUDE_DIRS}
+    ${MRML_INCLUDE_DIRS}
     ${vtkITK_INCLUDE_DIRS}
     ${SlicerBaseCLI_SOURCE_DIR}
     ${SlicerBaseCLI_BINARY_DIR}
-    ${vtkSlicerMarkupsModuleMRML_INCLUDE_DIRS}
+    ${Slicer_INSTALL_QTLOADABLEMODULES_INCLUDE_DIR}/vtkSlicerMarkupsModuleMRML
+    ${vtkAddon_INCLUDE_DIRS}
+  INSTALL_RUNTIME_DESTINATION ${Slicer_INSTALL_CLIMODULES_BIN_DIR}
+  INSTALL_LIBRARY_DESTINATION ${Slicer_INSTALL_CLIMODULES_LIB_DIR}
+  INSTALL_ARCHIVE_DESTINATION ${Slicer_INSTALL_CLIMODULES_LIB_DIR}
   )
 
+set_target_properties(${MODULE_NAME}Lib PROPERTIES
+  INSTALL_RPATH
+  "${Slicer_INSTALL_QTLOADABLEMODULES_LIB_DIR}")
+
 #-----------------------------------------------------------------------------
 if(BUILD_TESTING)
   add_subdirectory(Testing)
-- 
2.39.2

