diff --git a/Modules/CLI/ACPCTransform/CMakeLists.txt b/Modules/CLI/ACPCTransform/CMakeLists.txt
index b938e5bb83..e5b19e8b5c 100644
--- a/Modules/CLI/ACPCTransform/CMakeLists.txt
+++ b/Modules/CLI/ACPCTransform/CMakeLists.txt
@@ -1,7 +1,14 @@
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+#-----------------------------------------------------------------------------
+project(ACPCTransform)
 
 #-----------------------------------------------------------------------------
 set(MODULE_NAME ACPCTransform)
 
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
+
 #-----------------------------------------------------------------------------
 
 #
@@ -10,6 +17,8 @@ set(MODULE_NAME ACPCTransform)
 find_package(SlicerExecutionModel REQUIRED)
 include(${SlicerExecutionModel_USE_FILE})
 
+link_directories(${Slicer_INSTALL_QTLOADABLEMODULES_LIB_DIR})
+
 #-----------------------------------------------------------------------------
 SEMMacroBuildCLI(
   NAME ${MODULE_NAME}
@@ -25,12 +34,14 @@ SEMMacroBuildCLI(
     ${VTK_LIBRARIES}
     vtkSlicerMarkupsModuleMRML
   INCLUDE_DIRECTORIES
+    ${Slicer_Base_INCLUDE_DIRS}
     ${vtkTeem_INCLUDE_DIRS}
-    ${MRMLCore_INCLUDE_DIRS}
+    ${MRML_INCLUDE_DIRS}
     ${vtkITK_INCLUDE_DIRS}
     ${SlicerBaseCLI_SOURCE_DIR}
     ${SlicerBaseCLI_BINARY_DIR}
-    ${vtkSlicerMarkupsModuleMRML_INCLUDE_DIRS}
+    ${Slicer_INSTALL_QTLOADABLEMODULES_INCLUDE_DIR}/vtkSlicerMarkupsModuleMRML
+    ${vtkAddon_INCLUDE_DIRS}
   )
 
 #-----------------------------------------------------------------------------
