From 5fddc8f4d09f3a4038feafb3c5043cfab88e26a1 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Fri, 21 Feb 2020 13:56:01 +0100
Subject: [PATCH 2/3] COMP: Adding MRMLLib link to EditorLib/Logic

---
 Modules/Scripted/EditorLib/Logic/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/Scripted/EditorLib/Logic/CMakeLists.txt b/Modules/Scripted/EditorLib/Logic/CMakeLists.txt
index 325011e14..e065ea51b 100644
--- a/Modules/Scripted/EditorLib/Logic/CMakeLists.txt
+++ b/Modules/Scripted/EditorLib/Logic/CMakeLists.txt
@@ -19,6 +19,7 @@ set(${KIT}_SRCS
 
 set(${KIT}_TARGET_LIBRARIES
   ${VTK_LIBRARIES}
+  MRMLLogic
   )
 
 #-----------------------------------------------------------------------------
-- 
2.24.1

