From fc227f2c1887f10d012f662fca07ebfb5f0c2008 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Fri, 21 Feb 2020 13:56:25 +0100
Subject: [PATCH 3/3] COMP: Fix python wrapping error

---
 Modules/Scripted/EditorLib/Logic/vtkImageErode.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Modules/Scripted/EditorLib/Logic/vtkImageErode.h b/Modules/Scripted/EditorLib/Logic/vtkImageErode.h
index 9d58a01b2..f1d00323a 100644
--- a/Modules/Scripted/EditorLib/Logic/vtkImageErode.h
+++ b/Modules/Scripted/EditorLib/Logic/vtkImageErode.h
@@ -37,6 +37,8 @@ public:
 protected:
   vtkImageErode();
   ~vtkImageErode() override;
+  vtkImageErode(const vtkImageErode&);
+  void operator=(const vtkImageErode&);
 
   float Background;
   float Foreground;
-- 
2.24.1

