From 1426cf016fbf184ec394f8af7dba1244fdd6e24a Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 15 Apr 2020 16:30:08 +0200
Subject: [PATCH 4/4] COMP: Remove redundant special functions on vtkImageErode

---
 Modules/Scripted/EditorLib/Logic/vtkImageErode.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Modules/Scripted/EditorLib/Logic/vtkImageErode.h b/Modules/Scripted/EditorLib/Logic/vtkImageErode.h
index 44c8b3a604..3cb8d85c52 100644
--- a/Modules/Scripted/EditorLib/Logic/vtkImageErode.h
+++ b/Modules/Scripted/EditorLib/Logic/vtkImageErode.h
@@ -37,8 +37,6 @@ public:
 protected:
   vtkImageErode();
   ~vtkImageErode() override;
-  vtkImageErode(const vtkImageErode&);
-  void operator=(const vtkImageErode&);
 
   float Background;
   float Foreground;
-- 
2.24.1

