From 187426a192d3722f87bbbd02f0ef4f69b4319f25 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Fri, 21 Feb 2020 09:54:31 +0100
Subject: [PATCH 2/3] COMP: Change destination dir for python files

---
 .../Segmentations/EditorEffects/Python/CMakeLists.txt       | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Modules/Loadable/Segmentations/EditorEffects/Python/CMakeLists.txt b/Modules/Loadable/Segmentations/EditorEffects/Python/CMakeLists.txt
index d0fe11cf1..3b49470c4 100644
--- a/Modules/Loadable/Segmentations/EditorEffects/Python/CMakeLists.txt
+++ b/Modules/Loadable/Segmentations/EditorEffects/Python/CMakeLists.txt
@@ -1,7 +1,7 @@
 #-----------------------------------------------------------------------------
 configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/SegmentEditorEffects.__init__.py.in
-  ${CMAKE_BINARY_DIR}/${Slicer_QTSCRIPTEDMODULES_LIB_DIR}/SegmentEditorEffects/__init__.py
+  ${CMAKE_CURRENT_BINARY_DIR}/SegmentEditorEffects/__init__.py
   )
 
 #-----------------------------------------------------------------------------
@@ -38,6 +38,6 @@ ctkMacroCompilePythonScript(
   TARGET_NAME SegmentEditorEffects
   SCRIPTS "${SegmentEditorEffects_PYTHON_SCRIPTS}"
   RESOURCES "${SegmentEditorEffects_PYTHON_RESOURCES}"
-  DESTINATION_DIR ${Slicer_BINARY_DIR}/${Slicer_QTSCRIPTEDMODULES_LIB_DIR}/SegmentEditorEffects
-  INSTALL_DIR ${Slicer_INSTALL_QTSCRIPTEDMODULES_LIB_DIR}/SegmentEditorEffects
+  DESTINATION_DIR ${CMAKE_CURRENT_BINARY_DIR}/SegmentEditorEffects
+  INSTALL_DIR ${Slicer_INSTALL_QTSCRIPTEDMODULES_LIB_DIR}
   )
-- 
2.24.1

