From 6e0e893750e142371a6e04c1f7d96ebe35b7de9b Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 14 Apr 2020 14:17:22 +0200
Subject: [PATCH 5/5] COMP: Remove redundant special-functions declarations

---
 .../Loadable/Annotations/Logic/vtkSlicerAnnotationModuleLogic.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Modules/Loadable/Annotations/Logic/vtkSlicerAnnotationModuleLogic.h b/Modules/Loadable/Annotations/Logic/vtkSlicerAnnotationModuleLogic.h
index b5c7b3f11c..af1772a3c9 100644
--- a/Modules/Loadable/Annotations/Logic/vtkSlicerAnnotationModuleLogic.h
+++ b/Modules/Loadable/Annotations/Logic/vtkSlicerAnnotationModuleLogic.h
@@ -277,8 +277,6 @@ protected:
 
   vtkSlicerAnnotationModuleLogic();
   ~vtkSlicerAnnotationModuleLogic() override;
-  vtkSlicerAnnotationModuleLogic(const vtkSlicerAnnotationModuleLogic&);
-  void operator=(const vtkSlicerAnnotationModuleLogic&);
 
   // Initialize listening to MRML events
   void SetMRMLSceneInternal(vtkMRMLScene * newScene) override;
-- 
2.24.1

