diff --git a/Modules/Scripted/SegmentEditor/CMakeLists.txt b/Modules/Scripted/SegmentEditor/CMakeLists.txt
index f1f583501..c9c578c06 100644
--- a/Modules/Scripted/SegmentEditor/CMakeLists.txt
+++ b/Modules/Scripted/SegmentEditor/CMakeLists.txt
@@ -1,3 +1,13 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(SegmentEditor)
+
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
+
+find_package(CTK REQUIRED)
+include(${CTK_USE_FILE})
+
 #-----------------------------------------------------------------------------
 set(MODULE_NAME SegmentEditor)
 
diff --git a/Modules/Scripted/SegmentEditor/SubjectHierarchyPlugins/CMakeLists.txt b/Modules/Scripted/SegmentEditor/SubjectHierarchyPlugins/CMakeLists.txt
index 832a2e183..3607850c8 100644
--- a/Modules/Scripted/SegmentEditor/SubjectHierarchyPlugins/CMakeLists.txt
+++ b/Modules/Scripted/SegmentEditor/SubjectHierarchyPlugins/CMakeLists.txt
@@ -9,7 +9,7 @@ ctkMacroCompilePythonScript(
   TARGET_NAME ${MODULE_NAME}SubjectHierarchyPlugins
   SCRIPTS "${${MODULE_NAME}SubjectHierarchyPlugins_PYTHON_SCRIPTS}"
   RESOURCES "${${MODULE_NAME}SubjectHierarchyPlugins_PYTHON_RESOURCES}"
-  DESTINATION_DIR ${Slicer_BINARY_DIR}/${Slicer_QTSCRIPTEDMODULES_LIB_DIR}/SubjectHierarchyPlugins
+  DESTINATION_DIR ${CMAKE_CURRENT_BINARY_DIR}/SubjectHierarchyPlugins
   INSTALL_DIR ${Slicer_INSTALL_QTSCRIPTEDMODULES_LIB_DIR}/SubjectHierarchyPlugins
   NO_INSTALL_SUBDIR
   )
