From 8450a9fc3a44557874f3a3be544bbd438c4ecba6 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Fri, 17 Mar 2023 13:56:21 +0100
Subject: [PATCH] ENH: Make VectorToScalarVolume a separate module

---
 .../Scripted/VectorToScalarVolume/CMakeLists.txt   | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/Modules/Scripted/VectorToScalarVolume/CMakeLists.txt b/Modules/Scripted/VectorToScalarVolume/CMakeLists.txt
index 158c9256ba..243e2597ae 100644
--- a/Modules/Scripted/VectorToScalarVolume/CMakeLists.txt
+++ b/Modules/Scripted/VectorToScalarVolume/CMakeLists.txt
@@ -1,5 +1,17 @@
 #-----------------------------------------------------------------------------
-set(MODULE_NAME VectorToScalarVolume)
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+#-----------------------------------------------------------------------------
+project(VectorToScalarVolume)
+
+#-----------------------------------------------------------------------------
+set(MODULE_NAME "VectorToScalarVolume")
+
+string(TOUPPER ${MODULE_NAME} MODULE_NAME_UPPER)
+
+#-----------------------------------------------------------------------------
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
 
 #-----------------------------------------------------------------------------
 set(MODULE_PYTHON_SCRIPTS
-- 
2.39.2

