diff --git a/Modules/CLI/ExtractSkeleton/CMakeLists.txt b/Modules/CLI/ExtractSkeleton/CMakeLists.txt
index 87f22cc02c..0c1a114cc8 100755
--- a/Modules/CLI/ExtractSkeleton/CMakeLists.txt
+++ b/Modules/CLI/ExtractSkeleton/CMakeLists.txt
@@ -1,7 +1,15 @@
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+#-----------------------------------------------------------------------------
+project(ExtractSkeleton)
 
 #-----------------------------------------------------------------------------
 set(MODULE_NAME ExtractSkeleton)
 
+#-----------------------------------------------------------------------------
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
+
 #-----------------------------------------------------------------------------
 
 #
@@ -15,8 +23,27 @@ include(${SlicerExecutionModel_USE_FILE})
 #
 set(${PROJECT_NAME}_ITK_COMPONENTS
   ITKIOImageBase
+  ITKIOBMP
+  ITKIOBioRad
+  ITKIOBruker
+  ITKIOGDCM
+  ITKIOGE
+  ITKIOGIPL
+  ITKIOHDF5
+  ITKIOJPEG
+  ITKIOJPEG2000
+  ITKIOLSM
+  ITKIOMGHI
+  ITKIOMRC
+  ITKIOMetaImage
+  ITKIONIFTI
+  ITKIONRRD
+  ITKIOPNG
+  ITKIOScanco
+  ITKIOStimulate
+  ITKIOTIF
   )
-find_package(ITK 4.6 COMPONENTS ${${PROJECT_NAME}_ITK_COMPONENTS} REQUIRED)
+
 if(ITK_VERSION VERSION_GREATER_EQUAL "5.3")
   foreach(factory_uc IN ITEMS "IMAGEIO" "MESHIO" "TRANSFORMIO")
     set(ITK_NO_${factory_uc}_FACTORY_REGISTER_MANAGER 1)
@@ -32,6 +59,8 @@ if(WIN32)
   add_definitions(-D_CRT_SECURE_NO_DEPRECATE)
 endif()
 
+link_directories(${Slicer_INSTALL_QTLOADABLEMODULES_LIB_DIR})
+
 #-----------------------------------------------------------------------------
 SEMMacroBuildCLI(
   NAME ${MODULE_NAME}
@@ -47,11 +76,22 @@ SEMMacroBuildCLI(
   TARGET_LIBRARIES ${ITK_LIBRARIES}
     MRMLCore
     vtkSlicerMarkupsModuleMRML
+    ${VTK_LIBRARIES}
+    vtkAddon
   INCLUDE_DIRECTORIES
-    ${MRMLCore_INCLUDE_DIRS}
-    ${vtkSlicerMarkupsModuleMRML_INCLUDE_DIRS}
+    ${MRML_INCLUDE_DIRS}
+    ${vtkAddon_INCLUDE_DIRS}
+    ${Slicer_INSTALL_QTLOADABLEMODULES_INCLUDE_DIR}/vtkSlicerMarkupsModuleMRML
+  INSTALL_RUNTIME_DESTINATION ${Slicer_INSTALL_CLIMODULES_BIN_DIR}
+  INSTALL_LIBRARY_DESTINATION ${Slicer_INSTALL_CLIMODULES_LIB_DIR}
+  INSTALL_ARCHIVE_DESTINATION ${Slicer_INSTALL_CLIMODULES_LIB_DIR}
   )
 
+
+set_target_properties(${MODULE_NAME}Lib PROPERTIES
+  INSTALL_RPATH
+  "${Slicer_INSTALL_QTLOADABLEMODULES_LIB_DIR}")
+
 #-----------------------------------------------------------------------------
 if(BUILD_TESTING)
   add_subdirectory(Testing)
