From 99c7acae84def84ada6440279618fa3a3efee55a Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Tue, 14 Mar 2023 12:08:03 +0100
Subject: [PATCH] ENH: Make DICOMPlugins a separate module

---
 Modules/Scripted/DICOMPlugins/CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Modules/Scripted/DICOMPlugins/CMakeLists.txt b/Modules/Scripted/DICOMPlugins/CMakeLists.txt
index d0aa8b33b9..e33709006e 100644
--- a/Modules/Scripted/DICOMPlugins/CMakeLists.txt
+++ b/Modules/Scripted/DICOMPlugins/CMakeLists.txt
@@ -1,3 +1,11 @@
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+#-----------------------------------------------------------------------------
+project(DICOMPlugins)
+
+#-----------------------------------------------------------------------------
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
 
 #-----------------------------------------------------------------------------
 set(MODULE_NAME DICOMPlugins)
-- 
2.39.2

