From f4af368471060baf9f3b42eda8b6922bc2e76762 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 5 Feb 2020 15:59:39 +0100
Subject: [PATCH 2/3] COMP: Remove install sentence for VMTK-Targets

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 64d1f60..649c71f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -283,7 +283,7 @@ if(VMTK_SCRIPTS_ENABLED)
   #   ADD_SUBDIRECTORY(distribution)
   # endif(VMTK_ENABLE_DISTRIBUTION)
 
-  install(EXPORT VMTK-Targets DESTINATION ${VMTK_BINARY_DIR})
+  #install(EXPORT VMTK-Targets DESTINATION ${VMTK_BINARY_DIR})
 
   get_property(_wrap_hierarchy_targets GLOBAL PROPERTY VMTK_WRAP_HIERARCHY_TARGETS)
   if(_wrap_hierarchy_targets)
-- 
2.24.1

