From 8ff7b47c743bc8c6a9ec0eabaf5ed7dbe7e7b0d8 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 6 Feb 2020 14:21:23 +0100
Subject: [PATCH 2/3] COMP: Remove ITKEigen3

---
 Modules/ThirdParty/Eigen3/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/ThirdParty/Eigen3/CMakeLists.txt b/Modules/ThirdParty/Eigen3/CMakeLists.txt
index 8dbbb86de6..3291adddd5 100644
--- a/Modules/ThirdParty/Eigen3/CMakeLists.txt
+++ b/Modules/ThirdParty/Eigen3/CMakeLists.txt
@@ -9,7 +9,7 @@ if(ITK_USE_SYSTEM_EIGEN)
 else()
   set(_eigen_itk_target ITKInternalEigen3::Eigen)
 endif()
-set(ITKEigen3_LIBRARIES ${_eigen_itk_target})
+# set(ITKEigen3_LIBRARIES ${_eigen_itk_target})
 # Assume that Eigen generates a Eigen3Config.cmake
 set(_Eigen3_min_version 3.3)
 
-- 
2.26.2

