From 778ac8cca644a2fec288b2f34f9d78417da9c11a Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 6 Feb 2020 14:45:03 +0100
Subject: [PATCH 3/3] ENH: Add ITKThickness3D as local module

---
 Modules/Remote/Thickness3D.remote.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/Remote/Thickness3D.remote.cmake b/Modules/Remote/Thickness3D.remote.cmake
index 5b7cd06367..d5a3ceb7a6 100644
--- a/Modules/Remote/Thickness3D.remote.cmake
+++ b/Modules/Remote/Thickness3D.remote.cmake
@@ -45,6 +45,6 @@
 itk_fetch_module(Thickness3D
   "Tools for 3D thickness measurement"
   MODULE_COMPLIANCE_LEVEL 2
-  GIT_REPOSITORY ${git_protocol}://github.com/InsightSoftwareConsortium/ITKThickness3D.git
+  GIT_REPOSITORY ${WORKDIR}/ITKThickness3D
   GIT_TAG c39a6c777584ac3919de7d0381e398d9ee673ee0
 )
-- 
2.26.2

