From dc19d058ff78364cafda6b6dfebc6628bad34270 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 10 Sep 2019 12:12:55 +0200
Subject: [PATCH] 001-ITK-5.1.0-Change_itkMGHImageIO_repo_to_local

---
 Modules/Remote/MGHIO.remote.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/Remote/MGHIO.remote.cmake b/Modules/Remote/MGHIO.remote.cmake
index 67cd526e02..8e5c743c65 100644
--- a/Modules/Remote/MGHIO.remote.cmake
+++ b/Modules/Remote/MGHIO.remote.cmake
@@ -1,5 +1,5 @@
 itk_fetch_module(MGHIO
   "MGHIO ImageIO plugin for ITK"
-  GIT_REPOSITORY ${git_protocol}://github.com/Slicer/itkMGHImageIO.git
+  GIT_REPOSITORY ${WORKDIR}/itkMGHImageIO
   GIT_TAG 9519c2c38d7ba0a6bf93182000224a7698fc1ca7
   )
-- 
2.21.0

