From 2725a967d6c1b7bd2dc410e88fb3908ec558d817 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 10 Sep 2019 06:50:30 +0200
Subject: [PATCH 1/2] ENH: Include missing files

---
 CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 97b4b983..ba8dcbda 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -245,6 +245,12 @@ foreach(file
   CMake/ctkFunctionGetGccVersion.cmake
   CMake/ctkFunctionGetCompilerVisibilityFlags.cmake
   CMake/ctkFunctionCompileSnippets.cmake
+  CMake/ctkFunctionGeneratePluginUseFile.cmake
+  CMake/ctkFunctionExtractPluginTargets.cmake
+  CMake/ctkFunctionGetAllPluginTargets.cmake
+  CMake/ctkFunctionGetTargetDependencies.cmake
+  CMake/ctkMacroSetupPlugins.cmake
+  CMake/ctkFunctionGetPluginDependencies.cmake
   )
   include(${file})
   install(FILES ${file} DESTINATION ${CTK_INSTALL_CMAKE_DIR} COMPONENT Development)
@@ -255,6 +261,8 @@ foreach(file
   CMake/ctkLinkerAsNeededFlagCheck.cmake
   CMake/ctk_compile_python_scripts.cmake.in
   CMake/CMakeFindDependencyMacro.cmake
+  CMake/ctkWrapPythonQt.py
+  CMake/ctkMacroWrapPythonQtModuleInit.cpp.in
   )
   install(FILES ${file} DESTINATION ${CTK_INSTALL_CMAKE_DIR} COMPONENT Development)
 endforeach()
-- 
2.24.1

