From 3b14d6c7e6e151b89bc75853d55f45ab6c43038a Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 18 Apr 2018 16:32:20 +0200
Subject: [PATCH] Add files to install

---
 CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 50d9ca9f..6a82a1e9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -244,6 +244,12 @@ foreach(file
   CMake/ctkFunctionGetGccVersion.cmake
   CMake/ctkFunctionGetCompilerVisibilityFlags.cmake
   CMake/ctkFunctionCompileSnippets.cmake
+  CMake/ctkFunctionGeneratePluginUseFile.cmake
+  CMake/ctkFunctionExtractPluginTargets.cmake
+  CMake/ctkFunctionGetAllPluginTargets.cmake
+  CMake/ctkFunctionGetTargetDependencies.cmake
+  CMake/ctkMacroSetupPlugins.cmake
+  CMake/ctkFunctionGetPluginDependencies.cmake
   )
   include(${file})
   install(FILES ${file} DESTINATION ${CTK_INSTALL_CMAKE_DIR} COMPONENT Development)
@@ -254,6 +260,8 @@ foreach(file
   CMake/ctkLinkerAsNeededFlagCheck.cmake
   CMake/ctk_compile_python_scripts.cmake.in
   CMake/CMakeFindDependencyMacro.cmake
+  CMake/ctkWrapPythonQt.py
+  CMake/ctkMacroWrapPythonQtModuleInit.cpp.in
   )
   install(FILES ${file} DESTINATION ${CTK_INSTALL_CMAKE_DIR} COMPONENT Development)
 endforeach()
-- 
2.16.1

