From 17af560bfaedc3ce800cece2b0f3b6ef6fd57833 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 1 Oct 2019 14:56:16 +0200
Subject: [PATCH 02/14] COMP: Fix link libraries in QTGUI

---
 Base/QTGUI/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Base/QTGUI/CMakeLists.txt b/Base/QTGUI/CMakeLists.txt
index 1fedfef0d..d08424975 100644
--- a/Base/QTGUI/CMakeLists.txt
+++ b/Base/QTGUI/CMakeLists.txt
@@ -4,6 +4,11 @@ project(qSlicerBaseQTGUI)
 # See CMake/SlicerMacroBuildBaseQtLibrary.cmake for details
 #
 
+#
+# ITK
+#
+find_package(ITK COMPONENTS ITKCommon REQUIRED)
+
 set(KIT_export_directive "Q_SLICER_BASE_QTGUI_EXPORT")
 
 # Additional directories to include
@@ -306,6 +311,7 @@ endif()
 set(KIT_target_libraries
   qSlicerBaseQTCore
   qMRMLWidgets
+  -lCTKImageProcessingITKCore
   )
 
 # Python wrap
-- 
2.21.0

