From c24a49720b98d15824376c14aa2f9e39fc02812c Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Fri, 13 Sep 2019 19:14:19 +0200
Subject: [PATCH] ENH: Adding ITK to Base/QtCore

---
 Base/QTCore/CMakeLists.txt | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Base/QTCore/CMakeLists.txt b/Base/QTCore/CMakeLists.txt
index cd62ab757..17d9ae0ea 100644
--- a/Base/QTCore/CMakeLists.txt
+++ b/Base/QTCore/CMakeLists.txt
@@ -4,6 +4,13 @@ project(qSlicerBaseQTCore)
 # Dependencies
 #
 
+#
+#ITK
+#
+find_package(ITK COMPONENTS ITKCommon REQUIRED)
+set(ITK_NO_IO_FACTORY_REGISTER_MANAGER 1) # See Libs/ITKFactoryRegistration/CMakeLists.txt
+include(${ITK_USE_FILE})
+
 #
 # DCMTK
 #
-- 
2.21.0

