From e64a28e026e6ac36a2bca80fb2daae01cd615f22 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 18 Sep 2019 09:43:58 +0200
Subject: [PATCH] ENH: Add conditionals for using ctkAppLauncherEnvironment

---
 Base/QTCore/qSlicerCoreApplication.cxx | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Base/QTCore/qSlicerCoreApplication.cxx b/Base/QTCore/qSlicerCoreApplication.cxx
index 5dd13799b..18eff7dc7 100644
--- a/Base/QTCore/qSlicerCoreApplication.cxx
+++ b/Base/QTCore/qSlicerCoreApplication.cxx
@@ -215,6 +215,7 @@ void qSlicerCoreApplicationPrivate::init()
 
   this->SlicerHome = this->discoverSlicerHomeDirectory();
 
+#ifdef Slicer_Use_CTKAPPLAUNCHER
   // Save the environment if no launcher is used (this is for example the case
   // on MacOSX when slicer is started from an install tree)
   if (ctkAppLauncherEnvironment::currentLevel() == 0)
@@ -263,6 +264,8 @@ void qSlicerCoreApplicationPrivate::init()
       }
     }
 
+#endif
+
 #ifdef Slicer_USE_PYTHONQT_WITH_OPENSSL
   if (!QSslSocket::supportsSsl())
     {
@@ -708,7 +711,11 @@ bool qSlicerCoreApplication::testAttribute(qSlicerCoreApplication::ApplicationAt
 //-----------------------------------------------------------------------------
 QProcessEnvironment qSlicerCoreApplication::startupEnvironment() const
 {
+#ifdef Slicer_Use_CTKAPPLAUNCHER
   return ctkAppLauncherEnvironment::environment(0);
+#else
+  return QProcessEnvironment();
+#endif
 }
 
 //-----------------------------------------------------------------------------
-- 
2.21.0

