From 41c272838f6d11a24bac4a13f2b7d75b269bd581 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Fri, 20 Sep 2019 13:51:03 +0200
Subject: [PATCH] ENH: Adding ITK libraries to vtkAddon

---
 Libs/vtkAddon/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Libs/vtkAddon/CMakeLists.txt b/Libs/vtkAddon/CMakeLists.txt
index 2b11c769a..36d185e15 100644
--- a/Libs/vtkAddon/CMakeLists.txt
+++ b/Libs/vtkAddon/CMakeLists.txt
@@ -22,6 +22,12 @@ find_package(VTK REQUIRED)
 include(${VTK_USE_FILE})
 set(vtkAddon_LIBS ${VTK_LIBRARIES})
 
+#
+# ITK
+#
+
+find_package(ITK COMPONENTS ITKCommon REQUIRED)
+set(vtkAddon_LIBS ${vtkAddon_LIBS} ${ITK_LIBRARIES})
 
 # --------------------------------------------------------------------------
 # Configure headers
-- 
2.21.0

