From 0a3af26fcce931d29a8730b02d1c08129115ed50 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 26 Sep 2019 08:28:28 +0200
Subject: [PATCH] ENH: Fix linking of libraries in Base/QTGUI

---
 Base/QTGUI/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Base/QTGUI/CMakeLists.txt b/Base/QTGUI/CMakeLists.txt
index 3d126cae2..dab9c734e 100644
--- a/Base/QTGUI/CMakeLists.txt
+++ b/Base/QTGUI/CMakeLists.txt
@@ -305,7 +305,8 @@ endif()
 # Additional Target libraries (QT libs already included)
 set(KIT_target_libraries
   qSlicerBaseQTCore
-  qMRMLWidgets
+  qMRMLWidgets 
+  -lCTKImageProcessingITKCore
   )
 
 # Python wrap
@@ -353,14 +354,13 @@ SlicerMacroBuildBaseQtLibrary(
 # Slicer icon engine
 #-----------------------------------------------------------------------------
 
-set(CTK_LIBRARIES CTKWidgets CTKImageProcessingITKCore)
 ctkMacroBuildQtIconEnginesPlugin(
   NAME qSlicerIconEnginePlugin
   EXPORT_DIRECTIVE "Q_SLICER_BASE_QTGUI_ICON_ENGINE_EXPORT"
   FOLDER "Core-Base"
   SRCS qSlicerIconEnginePlugin.cxx qSlicerIconEnginePlugin.h
   MOC_SRCS qSlicerIconEnginePlugin.h
-  TARGET_LIBRARIES ${QT_LIBRARIES} ${CTK_LIBRARIES}
+  TARGET_LIBRARIES ${QT_LIBRARIES} CTKWidgets
   )
 
 #-----------------------------------------------------------------------------
-- 
2.21.0

