From f226de0e3fa12698aa0d1886d71add9ba3bebc43 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 15 Jan 2020 12:25:55 +0100
Subject: [PATCH] COMP: Make ModelMaker a separate module

---
 Modules/CLI/ModelMaker/CMakeLists.txt | 26 ++++++++++++++++++++++----
 1 file changed, 22 insertions(+), 4 deletions(-)

diff --git a/Modules/CLI/ModelMaker/CMakeLists.txt b/Modules/CLI/ModelMaker/CMakeLists.txt
index 9a455a619..6afe8ec85 100644
--- a/Modules/CLI/ModelMaker/CMakeLists.txt
+++ b/Modules/CLI/ModelMaker/CMakeLists.txt
@@ -1,25 +1,43 @@
+cmake_minimum_required(VERSION 3.0.2)
 
 #-----------------------------------------------------------------------------
 set(MODULE_NAME ModelMaker)
 
 #-----------------------------------------------------------------------------
 
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
 #
 # SlicerExecutionModel
 #
 find_package(SlicerExecutionModel REQUIRED)
 include(${SlicerExecutionModel_USE_FILE})
 
+find_package(VTK REQUIRED)
+include(${VTK_USE_FILE})
+
+find_package(ITK REQUIRED)
+include (${ITK_USE_FILE})
+
+link_directories(
+  ${Slicer_Libs_LIBRARY_DIRS}
+  ${Slicer_Base_LIBRARY_DIRS}
+  )
+
 #-----------------------------------------------------------------------------
 # TODO: Remove SlicerBaseLogic once pass in color node information
 SEMMacroBuildCLI(
   NAME ${MODULE_NAME}
   LOGO_HEADER ${Slicer_SOURCE_DIR}/Resources/NAMICLogo.h
-  TARGET_LIBRARIES vtkITK ModuleDescriptionParser MRMLCore SlicerBaseLogic SlicerBaseCLI ${VTK_LIBRARIES}
+  INSTALL_RUNTIME_DESTINATION ${Slicer_INSTALL_CLIMODULES_BIN_DIR}
+  INSTALL_LIBRARY_DESTINATION ${Slicer_INSTALL_CLIMODULES_LIB_DIR}
+  INSTALL_ARCHIVE_DESTINATION ${Slicer_INSTALL_CLIMODULES_SHARE_DIR}
+  TARGET_LIBRARIES vtkITK ModuleDescriptionParser MRMLCore SlicerBaseLogic SlicerBaseCLI ${VTK_LIBRARIES} ${ITK_LIBRARIES}
   INCLUDE_DIRECTORIES
-    ${vtkITK_INCLUDE_DIRS}
-    ${MRMLCore_INCLUDE_DIRS}
-    ${SlicerBaseLogic_SOURCE_DIR} ${SlicerBaseLogic_BINARY_DIR}
+  ${Slicer_Base_INCLUDE_DIRS}
+  ${vtkITK_INCLUDE_DIRS}
+  ${Slicer_Libs_INCLUDE_DIRS}
+  ${SlicerBaseLogic_SOURCE_DIR} ${SlicerBaseLogic_BINARY_DIR}
   )
 
 #-----------------------------------------------------------------------------
-- 
2.24.1

