From 68ca571f396c220413fb46a7b381e80edb02fc4c Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 14 Apr 2020 14:47:43 +0200
Subject: [PATCH 3/3] COMP: Remove redundant declaration of special functions

---
 .../Loadable/SceneViews/Logic/vtkSlicerSceneViewsModuleLogic.h  | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Modules/Loadable/SceneViews/Logic/vtkSlicerSceneViewsModuleLogic.h b/Modules/Loadable/SceneViews/Logic/vtkSlicerSceneViewsModuleLogic.h
index bcd4cdce1e..9566fd6e41 100644
--- a/Modules/Loadable/SceneViews/Logic/vtkSlicerSceneViewsModuleLogic.h
+++ b/Modules/Loadable/SceneViews/Logic/vtkSlicerSceneViewsModuleLogic.h
@@ -93,8 +93,6 @@ protected:
 
   vtkSlicerSceneViewsModuleLogic();
   ~vtkSlicerSceneViewsModuleLogic() override;
-  vtkSlicerSceneViewsModuleLogic(const vtkSlicerSceneViewsModuleLogic&);
-  void operator=(const vtkSlicerSceneViewsModuleLogic&);
 
   void OnMRMLSceneNodeAdded(vtkMRMLNode* node) override;
   void OnMRMLSceneEndImport() override;
-- 
2.24.1

