From 4482bae8135140943f50f51092e5175f282c7ed2 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Wed, 22 Feb 2023 17:45:06 +0100
Subject: [PATCH] ENH: Make AddScalarVolumes a separate module

---
 Modules/CLI/AddScalarVolumes/CMakeLists.txt | 33 ++++++++++++++++-----
 1 file changed, 25 insertions(+), 8 deletions(-)

diff --git a/Modules/CLI/AddScalarVolumes/CMakeLists.txt b/Modules/CLI/AddScalarVolumes/CMakeLists.txt
index 03512ce284..a41eb6ed4d 100644
--- a/Modules/CLI/AddScalarVolumes/CMakeLists.txt
+++ b/Modules/CLI/AddScalarVolumes/CMakeLists.txt
@@ -1,7 +1,15 @@
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+#-----------------------------------------------------------------------------
+project(AddScalarVolumes)
 
 #-----------------------------------------------------------------------------
 set(MODULE_NAME AddScalarVolumes)
 
+#-----------------------------------------------------------------------------
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
+
 #-----------------------------------------------------------------------------
 
 #
@@ -13,13 +21,7 @@ include(${SlicerExecutionModel_USE_FILE})
 #
 # ITK
 #
-set(${PROJECT_NAME}_ITK_COMPONENTS
-  ITKIOImageBase
-  ITKImageFunction
-  ITKImageGrid
-  ITKImageIntensity
-  )
-find_package(ITK 4.6 COMPONENTS ${${PROJECT_NAME}_ITK_COMPONENTS} REQUIRED)
+#
 if(ITK_VERSION VERSION_GREATER_EQUAL "5.3")
   foreach(factory_uc IN ITEMS "IMAGEIO" "MESHIO" "TRANSFORMIO")
     set(ITK_NO_${factory_uc}_FACTORY_REGISTER_MANAGER 1)
@@ -29,13 +31,28 @@ else()
 endif()
 include(${ITK_USE_FILE})
 
+set(${PROJECT_NAME}_INCLUDE_DIRECTORIES
+  ${Slicer_Base_INCLUDE_DIRS}
+)
 #-----------------------------------------------------------------------------
 SEMMacroBuildCLI(
   NAME ${MODULE_NAME}
   LOGO_HEADER ${Slicer_SOURCE_DIR}/Resources/ITKLogo.h
-  TARGET_LIBRARIES ${ITK_LIBRARIES}
+  TARGET_LIBRARIES ${ITK_LIBRARIES} ${Slice_Base_LIBRARIES}
+  INCLUDE_DIRECTORIES ${${PROJECT_NAME}_INCLUDE_DIRECTORIES}
+  INSTALL_RUNTIME_DESTINATION ${Slicer_INSTALL_CLIMODULES_BIN_DIR}
+  INSTALL_LIBRARY_DESTINATION ${Slicer_INSTALL_CLIMODULES_LIB_DIR}
+  INSTALL_ARCHIVE_DESTINATION ${Slicer_INSTALL_CLIMODULES_LIB_DIR}
   )
 
+set_target_properties(${MODULE_NAME}Lib PROPERTIES
+  INSTALL_RPATH
+  "")
+
+set_target_properties(${MODULE_NAME}Lib PROPERTIES
+  INSTALL_RPATH
+  "${Slicer_INSTALL_QTLOADABLEMODULES_LIB_DIR}")
+
 #-----------------------------------------------------------------------------
 if(BUILD_TESTING)
   add_subdirectory(Testing)
-- 
2.39.2

