From 6c6665b8edd48ac6253121dfcf3da10c0cad3874 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.paloma@rr-research.no>
Date: Wed, 18 Sep 2019 08:43:00 +0200
Subject: [PATCH] ENH: Include vtkEventData.h in module headers (vtkCommon)

---
 Common/Core/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Common/Core/CMakeLists.txt b/Common/Core/CMakeLists.txt
index 4a680d3837..192c987ef9 100644
--- a/Common/Core/CMakeLists.txt
+++ b/Common/Core/CMakeLists.txt
@@ -226,6 +226,7 @@ set(${vtk-module}_HDRS
   vtkDataArrayAccessor.h
   vtkDataArrayIteratorMacro.h
   vtkDataArrayTemplate.h
+  vtkEventData.h
   vtkGenericDataArrayLookupHelper.h
   vtkIOStream.h
   vtkIOStreamFwd.h
-- 
2.21.0

