From 195b201fd0d332cc7b0a1640cf5418bcef6786fc Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Fri, 21 Feb 2020 20:17:52 +0100
Subject: [PATCH] EHN: Make LabelStatistics scripted module a separate module

---
 Modules/Scripted/LabelStatistics/CMakeLists.txt | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/Modules/Scripted/LabelStatistics/CMakeLists.txt b/Modules/Scripted/LabelStatistics/CMakeLists.txt
index f006e1aaa..7fb750852 100644
--- a/Modules/Scripted/LabelStatistics/CMakeLists.txt
+++ b/Modules/Scripted/LabelStatistics/CMakeLists.txt
@@ -1,5 +1,15 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(LabelStatistics)
+
+find_package(Slicer REQUIRED)
+include(${Slicer_USE_FILE})
+
+find_package(CTK REQUIRED)
+include(${CTK_USE_FILE})
+
 #-----------------------------------------------------------------------------
-set(MODULE_NAME LabelStatistics)
+Set(MODULE_NAME LabelStatistics)
 
 #-----------------------------------------------------------------------------
 set(MODULE_PYTHON_SCRIPTS
-- 
2.24.1

