From 6fb37faa3bef0b5b8dd2dd2a918f1e1580b7c1c4 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Thu, 26 Jan 2023 07:54:56 +0100
Subject: [PATCH 4/5] Change CMake files installation path

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 519b2cd6..115bca5a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -559,7 +559,7 @@ endif()
 set(Teem_CV_LIBRARY_DEPENDS_FILE "\${Teem_ROOT_DIR}/${Teem_LIB_INSTALL_DIR}/${EXTRA_INSTALL_PATH}/TeemLibraryDepends.cmake")
 set(Teem_CV_EXECUTABLE_DIRS "\${Teem_ROOT_DIR}/bin")
 set(Teem_CV_LIBRARY_DIRS "\${Teem_ROOT_DIR}/${Teem_LIB_INSTALL_DIR}/${EXTRA_INSTALL_PATH}")
-set(Teem_CV_USE_FILE "\${Teem_ROOT_DIR}/${Teem_LIB_INSTALL_DIR}/${EXTRA_INSTALL_PATH}/TeemUse.cmake")
+set(Teem_CV_USE_FILE "${Teem_ROOT_DIR}/${Teem_SHARE_INSTALL_DIR}/Teem-${Teem_VERSION_STRING}/cmake/TeemUse.cmake")
 set(Teem_CV_INCLUDE_DIRS "\${Teem_ROOT_DIR}/include")
 set(Teem_CV_BUILT_LIBRARIES teem) # The libraries built by teem.  Currently we only build the mega library.
 
@@ -586,7 +586,7 @@ install(FILES
   "${Teem_SOURCE_DIR}/CMake/TeemUse.cmake"
   "${Teem_BINARY_DIR}/TeemBuildSettings.cmake"
   "${Teem_BINARY_DIR}/TeemLibraryDepends.cmake"
-  DESTINATION ${Teem_LIB_INSTALL_DIR}${EXTRA_INSTALL_PATH}
+  DESTINATION "${Teem_SHARE_INSTALL_DIR}/Teem-${Teem_VERSION_STRING}/cmake"
   COMPONENT Development
   )
 endif()
-- 
2.34.1

