From 4d0602f97cba49e112dff051cff9924543c0edd2 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 1 Oct 2019 14:45:07 +0200
Subject: [PATCH 01/30] COMP: Remove uneccessary link libraries for QTCore

---
 Base/QTCore/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Base/QTCore/CMakeLists.txt b/Base/QTCore/CMakeLists.txt
index e75b4d91fa..f33ff30a1f 100644
--- a/Base/QTCore/CMakeLists.txt
+++ b/Base/QTCore/CMakeLists.txt
@@ -170,9 +170,7 @@ set(KIT_target_libraries
   MRMLDisplayableManager
   ${CTKAppLauncherLib_LIBRARIES}
   CTKCore
-  CTKImageProcessingITKCore
   CTKVisualizationVTKCore
-  ${ITK_LIBRARIES}
   )
 
 if(Slicer_BUILD_DICOM_SUPPORT)
-- 
2.26.2

