From 6055be06ec0970ddc5c1099a6f685247d3fa0a6b Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Tue, 24 Jan 2023 09:51:51 +0100
Subject: [PATCH 01/28] COMP: Add vtk CommonSystem component as requirement

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4738ca1f73..fa278fdff3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -564,6 +564,7 @@ message(STATUS "  Slicer_VTK_VERSION_MAJOR is ${Slicer_VTK_VERSION_MAJOR}")
 # Slicer_VTK_COMPONENTS
 #-----------------------------------------------------------------------------
 set(Slicer_VTK_COMPONENTS
+  CommonSystem
   FiltersExtraction
   FiltersFlowPaths
   FiltersGeometry
-- 
2.39.2

