From 592c8fef4a146b60db9e730fc24c10a93ada363a Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 1 Oct 2019 14:45:07 +0200
Subject: [PATCH 01/29] COMP: Remove uneccessary link libraries for QTCore

---
 Base/QTCore/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Base/QTCore/CMakeLists.txt b/Base/QTCore/CMakeLists.txt
index 6c55787ee8..4930d90560 100644
--- a/Base/QTCore/CMakeLists.txt
+++ b/Base/QTCore/CMakeLists.txt
@@ -171,9 +171,7 @@ set(KIT_target_libraries
   MRMLDisplayableManager
   ${CTKAppLauncherLib_LIBRARIES}
   CTKCore
-  CTKImageProcessingITKCore
   CTKVisualizationVTKCore
-  ${ITK_LIBRARIES}
   )
 
 if(Slicer_BUILD_DICOM_SUPPORT)
-- 
2.26.2

