From f83105d62059120a277edfc803462296ba67fdf7 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 1 Oct 2019 14:56:16 +0200
Subject: [PATCH 02/29] COMP: Fix link libraries in QTGUI

---
 Base/QTGUI/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Base/QTGUI/CMakeLists.txt b/Base/QTGUI/CMakeLists.txt
index e7752a745e..f2133ac153 100644
--- a/Base/QTGUI/CMakeLists.txt
+++ b/Base/QTGUI/CMakeLists.txt
@@ -4,6 +4,11 @@ project(qSlicerBaseQTGUI)
 # See CMake/SlicerMacroBuildBaseQtLibrary.cmake for details
 #
 
+#
+# ITK
+#
+find_package(ITK COMPONENTS ITKCommon REQUIRED)
+
 set(KIT_export_directive "Q_SLICER_BASE_QTGUI_EXPORT")
 
 # Additional directories to include
@@ -308,6 +313,7 @@ endif()
 set(KIT_target_libraries
   qSlicerBaseQTCore
   qMRMLWidgets
+  -lCTKImageProcessingITKCore
   )
 
 # Python wrap
-- 
2.24.1

