diff --git a/Base/QTCore/CMakeLists.txt b/Base/QTCore/CMakeLists.txt
index 0c9aead4d..ceaf125ae 100644
--- a/Base/QTCore/CMakeLists.txt
+++ b/Base/QTCore/CMakeLists.txt
@@ -41,9 +41,8 @@ endif()
 # qRestAPI
 #
 if(Slicer_BUILD_EXTENSIONMANAGER_SUPPORT OR Slicer_BUILD_APPLICATIONUPDATE_SUPPORT)
-  find_package(qRestAPI REQUIRED)
-  include(${qRestAPI_USE_FILE})
-endif()
+ find_package(qRestAPI REQUIRED)
+ endif()
 
 #
 # CTKAppLauncherLib
