diff --git a/Base/QTCore/qSlicerCoreApplication.cxx b/Base/QTCore/qSlicerCoreApplication.cxx
index d3b4cc76a..7fe1d8871 100644
--- a/Base/QTCore/qSlicerCoreApplication.cxx
+++ b/Base/QTCore/qSlicerCoreApplication.cxx
@@ -2082,7 +2082,7 @@ bool qSlicerCoreApplication::loadCaCertificates(const QString& caCertificatesPat
     }
   return !QSslSocket::defaultCaCertificates().empty();
 #else
-  Q_UNUSED(slicerHome);
+  Q_UNUSED(caCertificatesPath);
   return false;
 #endif
 }
