From 63ff5e26b97f806d597a7a8b524903c1f66fa73f Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Wed, 25 Jan 2023 11:08:38 +0100
Subject: [PATCH 08/11] ENH: Remove the 'App-real' suffix from Slicer
 executable

---
 CMake/SlicerMacroBuildApplication.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/SlicerMacroBuildApplication.cmake b/CMake/SlicerMacroBuildApplication.cmake
index 76f643b694..3c142e051e 100644
--- a/CMake/SlicerMacroBuildApplication.cmake
+++ b/CMake/SlicerMacroBuildApplication.cmake
@@ -428,7 +428,7 @@ macro(slicerMacroBuildApplication)
   endif()
 
   set(executable_name ${SLICERAPP_APPLICATION_NAME})
-  if(NOT APPLE)
+  if(NOT APPLE AND Slicer_SUPERBUILD)
     set(executable_name ${executable_name}App-real)
   endif()
   message(STATUS "Setting ${SLICERAPP_APPLICATION_NAME} executable name to '${executable_name}${CMAKE_EXECUTABLE_SUFFIX}'")
-- 
2.34.1

