From d7cb31cc00debf5acb761d79c2f041783166cf53 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Mon, 7 Oct 2019 11:47:42 +0200
Subject: [PATCH 15/19] COMP: Adding conditional for installation of QT
 designer launcher

---
 CMake/SlicerCPack.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/SlicerCPack.cmake b/CMake/SlicerCPack.cmake
index ee2c2bb68..611252621 100644
--- a/CMake/SlicerCPack.cmake
+++ b/CMake/SlicerCPack.cmake
@@ -64,7 +64,7 @@ if(Slicer_BUILD_DICOM_SUPPORT AND NOT Slicer_USE_SYSTEM_DCMTK)
 endif()
 
 # Install Qt designer launcher
-if(Slicer_BUILD_QT_DESIGNER_PLUGINS)
+if(Slicer_BUILD_QT_DESIGNER_PLUGINS AND Slicer_USE_CTKAPPLAUNCHER)
   set(executablename "SlicerDesigner")
   set(build_designer_executable "${QT_BINARY_DIR}/designer${CMAKE_EXECUTABLE_SUFFIX}")
   if(APPLE)
-- 
2.24.1

