From 38c3121c7c529010bfbebbe5a35595a2ba129c41 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 15 Jan 2020 10:50:12 +0100
Subject: [PATCH 16/20] COMP: Change GLOB filter for installing vtkITK dev
 components

---
 Libs/vtkITK/CMakeLists.txt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Libs/vtkITK/CMakeLists.txt b/Libs/vtkITK/CMakeLists.txt
index abcd499f0..9e7106e19 100644
--- a/Libs/vtkITK/CMakeLists.txt
+++ b/Libs/vtkITK/CMakeLists.txt
@@ -83,10 +83,15 @@ if(NOT DEFINED ${PROJECT_NAME}_INSTALL_NO_DEVELOPMENT)
   set(${PROJECT_NAME}_INSTALL_NO_DEVELOPMENT ${Slicer_INSTALL_NO_DEVELOPMENT})
 endif()
 if(NOT ${PROJECT_NAME}_INSTALL_NO_DEVELOPMENT)
-  file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.(h|txx)")
+  file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
   install(
     FILES ${headers} ${CMAKE_CURRENT_BINARY_DIR}/${configure_header_file}
     DESTINATION include/${PROJECT_NAME} COMPONENT Development)
+
+  file(GLOB templates "${CMAKE_CURRENT_SOURCE_DIR}/*.txx")
+  install(
+    FILES ${templates}
+    DESTINATION include/${PROJECT_NAME} COMPONENT Development)
 endif()
 
 # --------------------------------------------------------------------------
-- 
2.24.1

