From df08a2c93a70b04521d03fca413fb3fcbfe0a5d7 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 15 Jan 2020 12:53:21 +0100
Subject: [PATCH] COMP: Add Slicer_USE_PYTHONQT as condition for ModuleLogic
 build macro

---
 CMake/SlicerMacroBuildModuleLogic.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/SlicerMacroBuildModuleLogic.cmake b/CMake/SlicerMacroBuildModuleLogic.cmake
index 2114d4fa1..449d83277 100644
--- a/CMake/SlicerMacroBuildModuleLogic.cmake
+++ b/CMake/SlicerMacroBuildModuleLogic.cmake
@@ -110,7 +110,7 @@ macro(SlicerMacroBuildModuleLogic)
   # --------------------------------------------------------------------------
   # Python wrapping
   # --------------------------------------------------------------------------
-  if(NOT ${MODULELOGIC_DISABLE_WRAP_PYTHON} AND VTK_WRAP_PYTHON AND BUILD_SHARED_LIBS)
+  if(Slicer_USE_PYTHONQT AND NOT ${MODULELOGIC_DISABLE_WRAP_PYTHON} AND VTK_WRAP_PYTHON AND BUILD_SHARED_LIBS)
 
     set(Slicer_Wrapped_LIBRARIES
       SlicerBaseLogicPythonD
-- 
2.24.1

