From b2d1f60fca3500d1b291b8f38bc5cf7525ec8c7f Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 19 Feb 2020 09:39:24 +0100
Subject: [PATCH 23/25] COMP: Set missing variables in SlicerConfig (install
 configuration)

---
 CMake/SlicerInstallConfig.cmake.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMake/SlicerInstallConfig.cmake.in b/CMake/SlicerInstallConfig.cmake.in
index e8f603047..eb3a8608a 100644
--- a/CMake/SlicerInstallConfig.cmake.in
+++ b/CMake/SlicerInstallConfig.cmake.in
@@ -7,6 +7,10 @@
 # or a Slicer installation.
 #
 
+# Slicer Libs VTK wrapped libraries
+set(Slicer_Libs_VTK_WRAPPED_LIBRARIES "@Slicer_Libs_VTK_WRAPPED_LIBRARIES@")
+set(PYTHON_INCLUDE_DIRS "@PYTHON_INCLUDE_DIRS@")
+
 # The C and C++ flags added by Slicer to the cmake-configured flags.
 set(Slicer_REQUIRED_C_FLAGS "@Slicer_REQUIRED_C_FLAGS@")
 set(Slicer_REQUIRED_CXX_FLAGS "@Slicer_REQUIRED_CXX_FLAGS@")
-- 
2.24.1

