From 50f673dce1df910c198739ca9f45682dd1386500 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 20 Feb 2020 15:07:42 +0100
Subject: [PATCH 24/25] COMP: Add needed include dirs for python wrapping of
 modules

---
 CMake/vtkMacroKitPythonWrap.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/vtkMacroKitPythonWrap.cmake b/CMake/vtkMacroKitPythonWrap.cmake
index 71e03ae64..5b4ba0667 100644
--- a/CMake/vtkMacroKitPythonWrap.cmake
+++ b/CMake/vtkMacroKitPythonWrap.cmake
@@ -143,7 +143,7 @@ macro(vtkMacroKitPythonWrap)
     # Add kit include dirs
     list(APPEND _kit_wrap_include_dirs ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
     if(DEFINED Slicer_Base_INCLUDE_DIRS)
-      list(APPEND _kit_wrap_include_dirs ${Slicer_Base_INCLUDE_DIRS})
+      list(APPEND _kit_wrap_include_dirs ${Slicer_Base_INCLUDE_DIRS} ${Slicer_Libs_INCLUDE_DIRS})
     endif()
     set(_kit_include_dirs ${${MY_KIT_NAME}_INCLUDE_DIRS})
     if(_kit_include_dirs)
-- 
2.24.1

