From 87f8efbb1d9fc648b7ea22b4aba3f604c359c061 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 20 May 2020 16:22:24 +0200
Subject: [PATCH 30/30] ENH: Adding ModuleWizard.py to the list of scripts
 compile/install

---
 Utilities/Scripts/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Utilities/Scripts/CMakeLists.txt b/Utilities/Scripts/CMakeLists.txt
index 519586584b..1958bf71ed 100644
--- a/Utilities/Scripts/CMakeLists.txt
+++ b/Utilities/Scripts/CMakeLists.txt
@@ -33,6 +33,7 @@ if(Slicer_USE_PYTHONQT)
   # Install Python libexec scripts
   set(Slicer_LIBEXEC_PYTHON_SCRIPTS
     ExtensionWizard.py
+    ModuleWizard.py
     )
 
   ctkMacroCompilePythonScript(
-- 
2.26.2

