diff --git a/Modules/Loadable/Markups/MRML/CMakeLists.txt b/Modules/Loadable/Markups/MRML/CMakeLists.txt
index 86720a9608..56b2e4fa6d 100644
--- a/Modules/Loadable/Markups/MRML/CMakeLists.txt
+++ b/Modules/Loadable/Markups/MRML/CMakeLists.txt
@@ -70,7 +70,7 @@ set(${KIT}_SRCS
   )
 
 set(${KIT}_TARGET_LIBRARIES
-  ${MRML_LIBRARIES}
+  ${ITK_LIBRARIES}
   )
 
 #-----------------------------------------------------------------------------
